From b73bc461774b7ab26eaf1fb6f6cb100cf4b06b07 Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Fri, 20 Jul 2018 11:25:17 +0100
Subject: [fix] hackerrank, 'mean var and std', python solution

-try python %g format which mixes %e (expo) and %f (float) formats
depending on target
---
 .../mean.var.and.std.py                                         | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hackerrank/python.-.numpy.-.easy.-.mean.var.and.std/mean.var.and.std.py b/hackerrank/python.-.numpy.-.easy.-.mean.var.and.std/mean.var.and.std.py
index ed51288..556687c 100644
--- a/hackerrank/python.-.numpy.-.easy.-.mean.var.and.std/mean.var.and.std.py
+++ b/hackerrank/python.-.numpy.-.easy.-.mean.var.and.std/mean.var.and.std.py
@@ -9,4 +9,4 @@ for _ in range(n):
 arr = numpy.array(l)
 print(numpy.mean(arr, axis = 1))
 print(numpy.var(arr, axis = 0))
-print("%.11f" % numpy.std(arr, axis = None))
+print("%.12g" % round(numpy.std(arr, axis = None), 12))
-- 
2.23.0

